File: test/fixtures/node-superagent/get_with_form.js
import request from 'superagent';

request
  .post('http://localhost:28139/v3')
  .auth('test', '')
  .field('from', 'test@tester.com')
  .field('to', 'devs@tester.net')
  .field('subject', 'Hello')
  .field('text', 'Testing the converter!')
  .then(res => {
     console.log(res.body);
  });

File: test/fixtures/ruby-httparty/get_with_form.rb
require 'httparty'

url = 'http://localhost:28139/v3'
auth = { username: 'test', password: ''}
body = {
  from: 'test@tester.com'
  to: 'devs@tester.net'
  subject: 'Hello'
  text: 'Testing the converter!'
}
res = HTTParty.post(url, basic_auth: auth, multipart: true, body: body)
